pipelines/batch/README.md

```diff
@@ -0,0 +1,193 @@
+  or an [OpenMRS](https://openmrs.org/) instance using MySQL. Note: JDBC support
+  beyond HAPI FHIR and OpenMRS is not currently planned. Our long-term approach
+  for a generic high-throughput alternative is to use the
+  [FHIR Bulk Export API](https://build.fhir.org/ig/HL7/bulk-data/export.html).
+
+There are two options for transforming the data (output):
+
+- _Parquet_: Outputs the FHIR resources as Parquet files, using the
+  [SQL-on-FHIR schema](https://github.com/FHIR/sql-on-fhir/blob/master/sql-on-fhir.md).
+- _FHIR_: Copies the FHIR resources to another FHIR server using FHIR APIs.
+
+## Setup
+
+1.  [Clone](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository)
+    the [FHIR Data Pipes](https://github.com/google/fhir-data-pipes) project to
+    your machine.
+1.  Set the `utils` directory to world-readable: `chmod -R 755 ./utils`.
+1.  Build binaries by running `mvn clean install` from the root directory of the
+    repository.
+
+## Run the pipeline
+
+Run the pipeline directly using the `java` command:
+
+```
+java -cp ./pipelines/batch/target/batch-bundled-0.1.0-SNAPSHOT.jar \
+    org.openmrs.analytics.FhirEtl \
+    --fhirServerUrl=http://example.org/fhir \
+    --[see additional parameters below]
+```
+
+Add the necessary parameters depending on your use case. The methods used for
+reading the source FHIR server and outputting the data depend on the parameters
+used. You can output to both Parquet files and a FHIR server by including the
+required parameters for both.
+
+## Parameters
+
+This section documents the parameters used by the various pipelines. For more
+information on parameters, see
+[`FhirEtlOptions`](https://github.com/google/fhir-data-pipes/blob/master/pipelines/batch/src/main/java/org/openmrs/analytics/FhirEtlOptions.java).
+
+### Common parameters
+
+These parameters are used regardless of other pipeline options.
+
+- `fhirServerUrl` - The base URL of the source FHIR server. Required.
+- `fhirServerUserName` - The HTTP Basic Auth username to access the FHIR server
+  APIs. Default: `admin`
+- `fhirServerPassword` - The HTTP Basic Auth password to access the FHIR server
+  APIs. Default: `Admin123`
+- `resourceList` - A comma-separated list of
+  [FHIR resources](https://www.hl7.org/fhir/resourcelist.html) to include in the
+  pipeline. Default: `Patient,Encounter,Observation`
+- `runner` -
+  [The Apache Beam Runner](https://beam.apache.org/documentation/runners/capability-matrix/)
+  to use. Pipelines supports DirectRunner and FlinkRunner by default; other
+  runners can be enabled by Maven profiles, e.g.,
+  [DataflowRunner](https://github.com/google/fhir-data-pipes/blob/16fcc255cef4d2708b9941a854e6c638b2533d45/pipelines/batch/pom.xml#L257).
+  Default: `DirectRunner`
+
+### FHIR-Search input parameters
+
+The pipeline will use FHIR-Search to fetch data as long as `jdbcModeEnabled` is
```

Comment:
Another update again related to PR #670: I have separated `--jdbcModeHapi` from `--jdbcModeEnabled`. So `--jdbcModeHapi` by itself is enough to enable JDBC mode for HAPI. We should probably rename `--jdbcModeEnabled` to `--jdbcModeOpenMRS`; WDYT?

Reply:
Makes sense; I'll update the docs as #670 is merged.